index.html

```diff
@@ -208,4 +208,4 @@ <h2 class="display-5">MADE WITH OPEN BRUSH</h2>
     integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
 </body>
 
-</html>
\ No newline at end of file
+</html>
```

Comment:
Please remove this hunk with the addition of a trailing newline :)

Reply:
Interesting, GitHub's web editor automatically adds that apparently. Fixed.